cmd/promtool/main.go

```diff
@@ -116,8 +116,27 @@ func checkConfig(t cli.Term, filename string) ([]string, error) {
 				return nil, err
 			}
 		}
-	}
 
+		for _, filesd := range scfg.ServiceDiscoveryConfig.FileSDConfigs {
+			for _, file := range filesd.Files {
+				// try to glob file, since it may be like /path/to/files/*.yml
```

Comment:
I'd actually remove the whole comment. It doesn't provide additional information on top of the code.